lib/checking_account.rb

```diff
@@ -0,0 +1,66 @@
+require 'csv'
+require_relative 'account'
+
+module Bank
+  class CheckingAccount < Bank::Account
+    attr_accessor :new_month
+    attr_reader :num_checks_used
+
+    def initialize(id, balance, open_date)
+      super(id, balance, open_date)
+      @num_checks_used = 0
+      @new_month = false
+    end
+
+    # Method that handles withdraw
+    def withdraw(withdraw_amount)
```

Comment:
Shouldn't you be able to use `super` in this method to avoid replicating existing functionality.